pkg/pillar/types/assignableadapters.go

```diff
@@ -98,6 +98,8 @@ type IoBundle struct {
 	Vfs sriov.VFList
 	// Only used in PhyIoNetEthVF
 	VfParams VfInfo
+	// Used for CAN interface properties
```

Comment:
is it really only can specific?

f.e. in `devmodel.pb.go` I see: "For example in WWAN to which firmware version to load etc"

Reply:
Actually nobody was using this field, it wasn't considered by pillar so far... but I agree, since it can be used for other purposes, it's better to generalize the comment....

Reply:
resolved

Reply:
resolved